esp/src/src-react/components/DFSExists.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService, WsDali } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+class MyDaliService extends DaliService {
+    DFSExists(request: WsDali.DFSExistsRequest): Promise<WsDali.ResultResponse> {
```

Comment:
This extended version of DaliService could be removed. The function added exists on the service being imported from hpcc-js/comms (https://github.com/hpcc-systems/Visualization/blob/trunk/packages/comms/src/services/wsdl/WsDali/v1.04/WsDali.ts#L152)